ChatSDK/ChatSDKFirebaseAdapter/Classes/Firebase/CCMessageWrapper.m

```diff
@@ -162,6 +163,11 @@ -(RXPromise *) deserialize: (NSDictionary *) value {
         //[_model setReadReceipts:readReceipts];
     }
     
+    NSDictionary * meta = value[bMetaDataPath];
```

Comment:
Why do you use `b_Meta` as the key above and then `bMetaDataPath` here?

Reply:
Fixed